pom.xml

```diff
@@ -166,6 +166,9 @@
                 <groupId>com.mycila</groupId>
                 <artifactId>license-maven-plugin</artifactId>
                 <configuration>
+                     <excludes>
+                        <exclude>src/test/resources/**</exclude>
```

Comment:
Instead of excluding the test html files, why not add the copyright header?

Reply:
@markmccallion28 Can you merge master branch into this branch so you consume the new OpenText copyright header?